src/components/WebexMember/WebexMember.jsx

```diff
@@ -20,6 +21,7 @@ export default function WebexMember({
   destinationID,
   personID,
   displayStatus,
+  showMe,
```

Comment:
This should not be pass as a prop but inferred by the data that is already in `WebexMember`